documentation/modules/rn-2.5.adoc

```diff
@@ -77,6 +80,10 @@ The validation service includes default validation rules for virtual machines fr
 
 The VMware vSphere source provider can now be created without specifying a VDDK `init` image. It is strongly recommended to create a VDDK `init` image to accelerate migrations.
 
+.Deployment on OKE enabled
+
+In {project-short} 2.5.3, deployment on OpenShift Kubernetes Engine (OKE) has been enabled. link:https://issues.redhat.com/browse/MTV-803[(MTV-803)]
```

Comment:
@ahadas - please advise what other details do you want to include here?

Reply:
I think that's good, we don't have to elaborate on OKE here. maybe adding a link to the documentation on OKE can be useful here, but I only found https://docs.openshift.com/container-platform/4.14/welcome/oke_about.html and I don't think we want to refer to a particular version of openshift